interface/forms/LBF/new.php

```diff
@@ -321,7 +321,7 @@ function end_row()
 ?>
 <html>
 <head>
-    <?php Header::setupHeader(['opener', 'common', 'datetime-picker', 'jquery-ui',]); ?>
+    <?php Header::setupHeader(['opener', 'common', 'datetime-picker']); ?>
```

Comment:
This file needs to be redone - please remove the following internal stylesheet styles:
- Line 328 - Remove `td, input, select, textarea` tag change style
- Line 333 - Remove `.table > tbody > tr > td` style change and add to the `<td>` element the `border-top-0` BS4 class
- Line 337 - The `div.section` style should be changed to this:
```
div.section {
            border: 1px solid var(--primary);
            margin: 0 0 0 0.8125rem;
            padding: 0.4375rem;
}
```

- Line 345 - remove `.form-control` class change style
- Line 351 - The `.RS` class style should be changed to this:
```
.RS {
            border-style: solid;
            border-width: 0 0 1px 0;
            border-color: var(--gray600);
}
```
- Line 357 - The `.RO` class style should be changed to this:
```
.RO {
     border-width: 1px solid var(--gray600) !important;
}
```